rules/bitstreams.bzl

```
@@ -0,0 +1,64 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+def _bitstreams_repo_impl(ctx):
```

minor nit: i think the proper convention is `repository_ctx` or `rctx` for repo rules, no?
```suggestion
def _bitstreams_repo_impl(rctx):
```

---

Done.